src/resources/dispatch/DispatchShow.tsx

```diff
@@ -192,19 +192,23 @@ export default function DispatchShow(): React.ReactElement {
   const dispatchAudits = async (itemId: Item['id']): Promise<void> => {
     const audiData = {
       type: AuditType.EDIT,
-      activityDetail: 'add item to dispatch',
+      activityDetail: `Dispatch Sent in ${record.reference}`,
       securityRelated: false,
       resource: constants.R_ITEMS,
       dataId: itemId
     }
     await audit(audiData)
-    await audit({
-      ...audiData,
-      resource: constants.R_DISPATCH
-    })
   }
 
   const dispatch = async (data: UpdateParams): Promise<void> => {
+    const audiData = {
+      type: AuditType.EDIT,
```

Comment:
could we introduce an AuditType.SENT please, and use that value in as the `type` please?